### PR TITLE
Pull request template improvements

### PR DESCRIPTION
**Pull request by dennishendriksen**
#### Checklist
- [ ] Unit tests
- [ ] Updated testplan

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 #### Checklist
-- [ ] Unit tests
-- [ ] Updated testplan
+- [ ] Functionality meets specifications
+- [ ] Functionality working
```
Comment (Mark-de-Haan) [inline .github/PULL_REQUEST_TEMPLATE.md]: Don't the specifications include functionality working?
Or is this a checkbox signalling there are no longer any bugs?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- [ ] Code reviewed
+- [ ] Code tested
```
Comment (Mark-de-Haan) [inline .github/PULL_REQUEST_TEMPLATE.md]: I guess the writer of the code can test, and the reviewer can test. Who is going to check the box?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+- [ ] User documentation updated
+- [ ] Test plan template updated
+- [ ] Clean commits
```
Comment (Mark-de-Haan) [inline .github/PULL_REQUEST_TEMPLATE.md]: This one is probably more obvious, but is the PR maker going to check this, or the reviewer?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
...
+- [ ] Build passes ('All checks have passed' in pull request)
```
Comment (Mark-de-Haan) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is also specified by a very big green √ at the bottom of a PR.
I don't think this needs to be in the template

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (bartcharbon) [inline .github/PULL_REQUEST_TEMPLATE.md]: is the green checkmark of the PR builder not the "code tested" check?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (bartcharbon) [inline .github/PULL_REQUEST_TEMPLATE.md]: agree with Mark, one checkbox would be enough (or I might be missing the added value of the second ;-) )

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (bartcharbon) [inline .github/PULL_REQUEST_TEMPLATE.md]: @Mark-de-Haan: The PR maker should take care of the clean commits, but the reviewer should review... same as the rest of the review...

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (tommydeboer) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think this is the "Unit tests added" checkbox

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (tommydeboer) [inline .github/PULL_REQUEST_TEMPLATE.md]: There can be bugs in a part of the functionality that wasn't detailed in the specifications. Alternatively, you can have working functionality that doesn't meet the specifications.

**Pull request merged**